bigbluebutton-html5/private/locales/en.json

```diff
@@ -126,8 +126,10 @@
     "app.meeting.meetingTimeRemaining": "Meeting time remaining: {0}",
     "app.meeting.meetingTimeHasEnded": "Time ended. Meeting will close soon",
     "app.meeting.endedMessage": "You will be forwarded back to the home screen",
-    "app.meeting.alertMeetingEndsUnderOneMinute": "Meeting is closing in a minute.",
-    "app.meeting.alertBreakoutEndsUnderOneMinute": "Breakout is closing in a minute.",
+    "app.meeting.alertMeetingEndsUnderMinutesSingular": "Meeting is closing in one minute.",
+    "app.meeting.alertMeetingEndsUnderMinutesPlural": "Meeting is closing in {0} minutes.",
+    "app.meeting.alertBreakoutEndsUnderMinutesSingular": "Breakout is closing in {0} minutes.",
```

Comment:
These 2 strings are flipped (Singular should be for one minute), otherwise looks good!